unfetter-discover/unfetter-store
# Issue 779/api security filter

supports unfetter-discover/unfetter#779

## Test
* Pull this feature branch w/ corresponding ui feature branch
* See ui PR for test instructions
* `cd unfetter-discover-api && npm run test`
* verify new security filter tests
